syndeo-template/pom.xml

```diff
@@ -217,6 +218,7 @@
         <dependency>
             <groupId>org.apache.beam</groupId>
             <artifactId>beam-sdks-java-extensions-google-cloud-platform-core</artifactId>
+            <version>${beam.snapshot.version}</version>
```

Comment:
nit: shouldn't use this if we are using the same version from the bom

Reply:
done